include/matrix.hpp

```diff
@@ -1,40 +1,68 @@
 #pragma once
 
 struct Matrix {
-    public:
-    Matrix(int rows_n,int columns_n){
+public:
+    Matrix(size_t rows_n, size_t columns_n){
```

Comment:
`size_t` è un tipo come `int`, `double`, etc...
è un numero unsigned (quindi positivo semi-definito) che is usa per indicare dimensioni